src/command/render/render-files.ts

```diff
@@ -475,17 +475,19 @@ export async function renderFiles(
           });
 
           // callback
-          pushTiming("render-pandoc");
-          await pandocRenderer.onRender(format, {
-            context,
-            recipe,
-            executeResult: unmappedExecuteResult!,
-            resourceFiles,
-          }, pandocQuiet);
-          popTiming();
+          try {
+            pushTiming("render-pandoc");
+            await pandocRenderer!.onRender(format, {
+              context,
+              recipe,
+              executeResult: unmappedExecuteResult!,
+              resourceFiles,
+            }, pandocQuiet);
+          } finally {
+            popTiming();
+          }
```

Comment:
(In case someone's looking at the diffs: this change is unrelated, but I'm positive it's a bug where we could be popping the timing information incorrectly if an exception was raised in an unexpected place.)